test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/security/ThrowablesTest.scala

```diff
@@ -0,0 +1,18 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.javalib.security
+
+import org.junit.Test
+
+class ThrowablesTest {
+  @Test def should_define_all_java_security_Errors_and_Exceptions(): Unit = {
+    import java.security._
+    new SecurityException("", new Exception())
```

Comment:
`SecurityException` should go in `org.scalajs.testsuite.javalib.lang.ThrowablesTest`.